yt/utilities/parallel_tools/parallel_analysis_interface.py

```diff
@@ -100,7 +100,9 @@ def enable_parallelism(suppress_logging: bool = False, communicator=None) -> boo
 
     # if no communicator specified, set to COMM_WORLD
     if communicator is None:
-        communicator = MPI.COMM_WORLD
+        communicator = MPI.COMM_WORLD.Dup()
+    else:
+        communicator = communicator.Dup()
```

Comment:
Semantically equivalent but avoid duplicating code (pun intended)
```suggestion
        communicator = MPI.COMM_WORLD
    communicator = communicator.Dup()
```

Reply:
Is it equivalent? In your version, `COMM_WORLD` will be Dup'ed twice which is not free of side-effects with MPI afaik.

Reply:
Really ? The `Dup` method is only called once

Reply:
I read too fast! My apologies :)